pyface/ui/qt4/system_metrics.py

```diff
@@ -40,12 +40,24 @@ class SystemMetrics(MSystemMetrics, HasTraits):
     # ------------------------------------------------------------------------
 
     def _get_screen_width(self):
-        return QtGui.QApplication.instance().desktop().screenGeometry().width()
+        # QDesktopWidget.screenGeometry() is deprecated and Qt docs
+        # suggest using screens() instead, but screens in not available in qt4
+        if QtCore.__version_info__ >= (5, 0):
```

Comment:
You might use `is_qt5` constant from `pyface.qt` here: https://github.com/enthought/pyface/blob/daa26ff0afbb36831be2febab6ab5fa591a2e47a/pyface/qt/__init__.py#L95